build.sbt

```diff
@@ -38,7 +37,8 @@ libraryDependencies ++= Seq(
   libraryDependencies ++= Seq(
     "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(excludeHadoop),
     "org.apache.spark" % "spark-mllib_2.10" % sparkVersion excludeAll(excludeHadoop),
-    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop)
+    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop),
+    "edu.berkeley.cs.amplab" % "mlmatrix_2.10" % "0.1" excludeAll(excludeHadoop)
```

Comment:
Now that I think about it its also better to exclude Spark here as we don't want to inherit Spark from ml-matrix. Can you add another exclusion rule above and add that here ? The syntax will be `excludeAll(excludeHadoop, excludeSpark)`